### ISSUE TITLE
Ability to select text across multiple blocks (mobile)

### ISSUE THREAD

**Issue by ognsya** [siyuan-note/siyuan#8900]
### In what scenarios do you need this feature?

I'd like to be able to select text across multiple blocks, including nested blocks.

For example, in the image shown in the comment below, I'd like to be able to select text from Parent all the way to Child 2.

### Describe the optimal solution

Allow text to be selected across multiple blocks.

### Describe the candidate solution

-

### Other information

I currently only have access to the mobile app. I'm not sure if the cross-block selection is possible on desktop or browser.

**Comment by ognsya** [comment]
![Screenshot_20230803-072438](https://github.com/siyuan-note/siyuan/assets/67518000/ab1b9865-ec66-4244-b690-087cee70cac2)

**Comment by ognsya** [comment]
I just tested on desktop browser. There I was able to select multiple blocks, however it is limited. I couldn't:

* select partial sub-trees. For example, in the situation shown in the picture above, I couldn't select only Parent and Child 1 (Child 2 gets selected as well). 
* select partial text across blocks. For example, there is no way to select text from the middle of a block until the middle of another.
